### ISSUE TITLE
Discuss Demand Probability for other Federal Agencies

### ISSUE THREAD

**Issue by joshbruce** [joshbruce/iae-uikit-php#5]
This issue is for the express purpose of discussing whether or not this seems like something other Federal Agency developers would be interested in using.

**Comment by joshbruce** [comment]
@mattlangan - I'm pretty old-school as well on front-end development. Then I moved to iOS a fair bit - where I was first introduced to the concept of a UI Kit for creating interfaces. It's similar to template engines within frameworks. The limitation I'm running into with using templates in frameworks is becoming dependent on the framework itself.

In other words, if I create a component in AngularJS's template engine and I move to something like NodeJS, I have to rewrite all of that. So, with the concept of a UI Kit for the web - specifically the USWDS - the mantra has become: Develop for the language, not the framework.

At a base level, I find this to make the most reusable concept. Because then I can configure the components from within whatever framework I am using - and, if the one I came from and to are in the same language - I don't need to rewrite anything.

**Comment by Drakenhart** [comment]
@joshbruce I agree with the idea of developing for the language not the framework. 

Also, I see some folks using .js to create the physical interface when simple html elements will suffice. I understand there are sometimes _needs_ for it, but even then CSS is supposed to be the design/visual portion of the three part build up (as I see it) to web interfaces.

If the UI pieces (not interaction-functional) can be built with as pure CSS as possible, it would be easier to use it between frameworks and languages. Staying away from hacks will be the tricky part because there are a LOT of things CSS can do if you mis-use it's power. (See: Accordions).

The same could be said with the other portions (back end code, front end interaction-functional code).

**Comment by joshbruce** [comment]
@Drakenhart - I'm not sure I'm following the JS being used to create the physical interface piece? Can you clarify?
